### DISCUSSION TITLE
Is there a way or method to control the distance of each scroll?

### DISCUSSION THREAD

**Discussion by ltaoo** [pichillilorenzo/flutter_inappwebview#2401] [Q&A] [answered]
I am experiencing an issue where the mouse scroll distance is too large when scrolling on Windows platform. The scroll sensitivity seems to be excessively high.

In Chrome
![Snipaste_2024-11-06_19-20-07](https://github.com/user-attachments/assets/64702374-747e-460f-924d-c67da84819e3)

In webview
![Snipaste_2024-11-06_19-19-39](https://github.com/user-attachments/assets/628f90c9-3626-45ea-b44e-ae3d4d4a5f69)

**Comment by pichillilorenzo** [comment, marked as answer]
Yes, with latest beta release, the scroll has been adjusted on Windows. Also, you can control the scroll multiplier on Windows with the new InAppWebViewSettings.scrollMultiplier parameter.

**Reply by ltaoo** [reply]
Thank you very much, this is exactly what I wanted.
